daemons/health-check-daemon/Makefile

```diff
@@ -2,7 +2,7 @@ include ../../common.mk
 .PHONY: install build stage deploy clobber
 
 ZIP_FILE=health_check_daemon.zip
-BUCKET=$(BUCKET_NAME_PREFIX)lambda-healthcheck-deployment-$(DEPLOYMENT_STAGE)
+BUCKET=$(BUCKET_NAME_PREFIX)lambda-deployment-$(DEPLOYMENT_STAGE)
 STAGED_FILE_KEY=health-check/$(ZIP_FILE)
```

Comment:
Extracting the name of something into a variable and then using that variable everywhere can reduce future copy-pasta errors.  In this case if "health_check_daemon" was in a variable, you could base the ZIP_FILE and STAGED_FILE_KEY off it.  The less times you have to enter the name of something in, the less room for error, and the more consistent it becomes.